config.yaml

```diff
@@ -1,12 +1,4 @@
-# YAML Config file for configuring the BSF-bot data directories to help the bot identify where to find data
-
-# Base directory for the BSF-bot-data submodule.
 "data-folder": "BSF-bot-data"
```

Comment:
Did all the comments get removed?

Reply:
I think I forgot to push the config changes for the original refactor/cogs branch? They should be saved somewhere on my local repository.